src/handlers/issue-matching.ts

```diff
@@ -0,0 +1,150 @@
+import { Context } from "../types";
+import { IssuePayload } from "../types/payload";
+
+export interface IssueGraphqlResponse {
+  node: {
+    title: string;
+    url: string;
+    state: string;
+    stateReason: string;
+    closed: boolean;
+    repository: {
+      owner: {
+        login: string;
+      };
+      name: string;
+    };
+    assignees: {
+      nodes: Array<{
+        login: string;
+        url: string;
+      }>;
+    };
+  };
+  similarity: number;
+}
+
+const commentBuilder = (matchResultArray: Map<string, Array<string>>): string => {
+  const commentLines: string[] = [">[!NOTE]", ">The following contributors may be suitable for this task:"];
+  matchResultArray.forEach((issues, assignee) => {
+    commentLines.push(`>### [${assignee}](https://www.github.com/${assignee})`);
```

Comment:
```suggestion
    commentLines.push(`> ### [${assignee}](https://www.github.com/${assignee})`);
```

Reply:
It works well both with and without the space, based on my tests.
>[!NOTE]
>Hello
>### Hello
> ### Hello